### PR TITLE
[WEB-3130] - Remove FE validation for minimum character count in MRN field

### PR DESCRIPTION
**Pull request by henry-tp**
[WEB-3130]

| Before | After |
|-----|-----|
| image | image |

[WEB-3130]: https://tidepool.atlassian.net/browse/WEB-3130?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by henry-tp** [comment]
For next sprint, but this seemed like something I could take on to start biting into the codebase

File: test/unit/pages/ClinicPatients.test.js [hunk 1]
```diff
@@ -796,12 +796,6 @@ describe('ClinicPatients', () => {
 
       expect(dialog().find('Button#addPatientConfirm').prop('disabled')).to.be.true;
 
-      expect(patientForm().find('input[name="mrn"]').prop('value')).to.equal('');
-      patientForm().find('input[name="mrn"]').simulate('change', { persist: noop, target: { name: 'mrn', value: 'mr2' } });
-      expect(patientForm().find('input[name="mrn"]').prop('value')).to.equal('MR2');
-
-      expect(dialog().find('Button#addPatientConfirm').prop('disabled')).to.be.true;
```
[the remark this answers is not in the document]
Comment (krystophv) [reply test/unit/pages/ClinicPatients.test.js:803]: I guess you could effectively leave the test in place, update the `change` to be a single character MRN and verify that the button is _not_ disabled?

[the remark this answers is not in the document]
Comment (henry-tp) [reply test/unit/pages/ClinicPatients.test.js:803]: Makes sense, re-added in 63565841eab43b10260389d8a29867c6fb73abd0

**Review by krystophv — APPROVED** [review]
LGTM 🚀

**Comment by henry-tp** [comment]
Closed in favor of #1474

**Pull request closed without merging**